pkg/plugins/common/kustomize/v1/scaffolds/internal/templates/config/rbac/leader_election_role.go

```diff
@@ -44,6 +45,13 @@ const leaderElectionRoleTemplate = `# permissions to do leader election.
 apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
 metadata:
+  labels:
+    app.kubernetes.io/name: role
+    app.kubernetes.io/instance: leader-election-role
+    app.kubernetes.io/component: rbac
+    app.kubernetes.io/created-by: {{ .ProjectName }}
+    app.kubernetes.io/part-of: {{ .ProjectName }}
+    app.kubernetes.io/managed-by: k8s.io
```

Comment:
```suggestion
    app.kubernetes.io/managed-by: kustomize
```

Reply:
resolved in commit: https://github.com/kubernetes-sigs/kubebuilder/pull/2800/commits/de5ad3beb0988a49f533ee8c1a8c87fd82e7bf02